### ISSUE TITLE
 [validation] The type attribute is unnecessary for JavaScript resources

### ISSUE THREAD

**Issue by UsmanAfzal** [WeAreSnook/OR-UK-frontend#161]
URL: https://openreferraluk.org/

The type attribute is unnecessary for JavaScript resources.
`<script type="text/javascript" async="" src="https://www.google-analytics.com/analytics.js"></script>`

**Comment by kingdomlevel** [comment]
Depends on #209 being merged as this is a ref to the analytics script.

**Comment by kingdomlevel** [comment]
PR #209 has now been merged - removing the 'blocked' label and moving to 'To Do'

**Comment by kingdomlevel** [comment]
I looked in to this today but unfortunately it can't be fixed by our side right now!

The Google Analytics tag is generated by Google Tag Manager which is controlled outwith the Snook organisation (maybe: by Porism whilst they hold the tenancy for the ORUK project?).

Even with access to Google Tag Manager I'm not sure if this will be possible to change.

Labelling as **Blocked** and **Talk to Porism** for now.

**Comment by MikeThacker1** [comment]
We believe the tag is created clientside by standard Google Tag Manager javascript. I will get Porism's Tag Manager expert to take a look. (This is Porism's Jira case https://porism.atlassian.net/browse/ORUK-91 which only Porism people are able to see).

@UsmanAfzal says the attribute is unnecessary. But is it in any way harmful?

**Comment by UsmanAfzal** [comment]
@MikeThacker1 This is low priority and not worth spending too much time on.

**Comment by MikeThacker1** [comment]
Our Google Tag Manager person says:

"It does appear that that script is being added by Tag Manager so we probably have little control over it. I will post a quick question on the GTM user forums to see if anyone has addressed it, but otherwise we may need to leave it in which case I’ll check what the implications are. It would seem odd that Google would leave it in without good reason."

**Comment by UsmanAfzal** [comment]
Possibly a legacy implementation, not using the type attribute is based on the HTML5 spec and the google docs use an example tag without the attribute. 

https://developers.google.com/analytics/devguides/collection/analyticsjs

**Comment by UsmanAfzal** [comment]
Happy to close as it is a nice to have and also have found it to be removed from the site.

**Comment by MikeThacker1** [comment]
This has been closed, but just to confirm what a Google Tag Manager (GTM) expert said on their support forum:
”Google Tag Manager deploys the analytics.js script as you say. The type attribute is presumably included because GTM doesn’t control the environment in which it is deployed. There is no way currently to omit the type attribute in the GTM configuration. If it is a big deal then you could bypass GTM and insert the analytics script directly but I would argue against it, since GTM is the gold standard method of integration Google Analytics and you will be trading massive flexibility and control through GTM for the resolution of a very minor validation fail.”
